workspaces/local-cli/package.json

```diff
@@ -43,7 +43,8 @@
     "tslib": "^1",
     "url-join": "^4.0.1",
     "uuid": "^8.0.0",
-    "which": "^2.0.2"
+    "which": "^2.0.2",
+    "human-readable-ids": "^1.0.4"
```

Comment:
if you run yarn install, it'll sort these dependencies. That will lead to less noise on diffs in the file.